### PR TITLE
BI-3927 Added route to enter-information page from confirm-company

### PR DESCRIPTION
**Pull request by SVarallo**

### CODE REVIEW

File: views/confirm-company.html [hunk 1]
```diff
@@ -78,11 +78,11 @@ <h2 class="govuk-heading-l">{{company.companyName}}</h2>
         ]
       }) }}
 
-      <form action="" method="post">
+      <form method="get" action="/strike-off-objections/enter-information">
```
Comment (markpit) [inline views/confirm-company.html:81]: forms should be submitted via a POST

Comment (SVarallo) [reply views/confirm-company.html:81]: This is how it's done in PTF and as far as the story goes we're not posting anything

Comment (mwestacott) [reply views/confirm-company.html:81]: @markpit might be thinking about the PATCH call to the API as we move to the next screen. I'm happy to consider making a change to POST when I integrate the call to the web stub, so this looks ok for now.

Comment (markpit) [reply views/confirm-company.html:81]: POST is better for security as GET translates form-data into url attributes viewable in teh browser. https://www.w3schools.com/tags/att_form_method.asp

Comment (mwestacott) [reply views/confirm-company.html:81]: We're not sending any form data at this point though. Are we?

Comment (markpit) [reply views/confirm-company.html:81]: the GET url places the attributes marker on '?' on the end of the url - doesn't look good if there are no attributes.
http://web.chs-dev.internal:4000/strike-off-objections/enter-information?

**Pull request merged**